packages/source-iotsitewise/src/time-series-data/types.ts

```diff
@@ -13,6 +17,11 @@ export type PropertyQuery = {
   refId?: RefId;
   resolution?: string;
   cacheSettings?: CacheSettings;
+  associatedStreams?: StreamAssociation[];
+  streamType?: StreamType;
+  alarmSource?: AlarmSource;
+  name?: string;
```

Comment:
We actually already have name and detailedName as part of styleSettings. Remove this.